qark/qarkMain.py

```diff
@@ -242,11 +244,10 @@ def list_all_apk():
     p0 = Popen([adb, 'shell', 'pm', 'list', 'packages', '-f'], stdout=PIPE, stdin=PIPE, stderr=STDOUT)
     index = 0
     for line in p0.stdout:
-
-
         path = str(line).find('=')
-        result.append(str(line)[8:path])
-        index+=1
+        if (path > 0):  # Check that a '=' was actually found - Ignores linker warnings etc. that ride along with the output
+            result.append(str(line)[8:path])
+            index+=1
```

Comment:
Before appending the path, make sure that it is in fact a path entry (Linker warnings etc. were being returned and inserted into the list).